infra/config-store/src/main/java/com/simprints/infra/config/store/ConfigRepositoryImpl.kt

```diff
@@ -36,6 +37,7 @@ internal class ConfigRepositoryImpl @Inject constructor(
     companion object {
         @VisibleForTesting
         const val PRIVACY_NOTICE_FILE = "privacy_notice"
+        private const val NOT_FOUND_STATUS_CODE = 404
```

Comment:
I would suggest using the codes defined in the HTTP library to skip on re-defining them in our code base.